zebedee-cms/src/main/java/com/github/onsdigital/zebedee/data/processing/DataIndex.java

```diff
@@ -57,12 +57,15 @@ public void setUriForCdid(String cdid, String uri) {
     public void reindex() {
         indexBuilt = false;
         Runnable build = () -> {
+            logInfo("Start building data index.").log();
+            long startTime = System.nanoTime();
             try {
                 Files.walkFileTree(contentReader.getRootFolder(), new IndexBuilder(index, contentReader));
             } catch (IOException e) {
-                logError(e);
+                logError(e, "Failed to build data index");
```

Comment:
you need to call `.log()` on the end of `logError`  there is also a `logAndThrow()` if you wanted to throw the exception.